### PR TITLE
Colin Narans

### PR DESCRIPTION
**Pull request by cnarans**

### CODE REVIEW

File: lib/numbers.md [hunk 1]
```diff
@@ -9,16 +9,16 @@ Fixnum for integers and Float for decimal numbers.
...
++, -, *, /, ** and % for addition, substraction, multiplication, division, exponential multiplication and modular division.  The most common comparisons are ==, !=, >, <, >= and <= for equals, does not equal, greater than, less than, greater than or equal to and less than or equal to.
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-The '+' operator produces the sum of two number, or concatenates two strings.
+The '+' operator produces the sum of two numbers, while it concatenates two strings.
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
 It will not work unless you convert it to another data type.  "20".to_i will convert the string to an integer.
```
Comment (halfghaninne) [inline lib/numbers.md]: What result to you get when you run `"20" * 3`? Does this operation still _work_, even if it doesn't work the way we'd expect/hope it to?

Comment (halfghaninne) [inline lib/numbers.md:20]: What result to you get when you run `"20" * 3`? Does this operation still _work_, even if it doesn't work the way we'd expect/hope it to?

If you do convert `"20"` to an Integer, then try to _divide_ it by 3, do you get a precise answer? How might you get a more precise answer?

**Comment by cnarans** [comment]
Yes, multiplying a string will repeat the string as many times as the multiple, so "20" \* 3 will give "202020".

Converting 20 to float will give a more precise answer, so you could do "20".to_f

**Comment by halfghaninne** [comment]
Nice work!

**Pull request closed without merging**